lib/date_and_time_methods.rb

```diff
@@ -8,7 +8,7 @@ def format_date_time(datetime, options = {})
   end
 
   def format_date(date)
-    date.strftime '%A, %B %e, %Y'
+    date.strftime '%A, %B %-d, %Y'
```

Comment:
I'd be curious to know why we need to change this generic method.

Reply:
@artarazavi may have wanted to write a better commit message, but it's something @werebus and I agreed on. When she ran the test suite, one of the feature tests failed depending on what day she ran it. There is an expectation that a date displayed on some page will match `record.date_for_later`. It should, but it doesn't, because it's looking for "Monday 6 2018" and instead finds "Monday  6 2018". Do you see the difference? It took me a while.
We brainstormed solutions to this, and determined that it was really unnecessary to display dates with two spaces in them IFF the date has one digit.

Reply:
Makes sense - thanks!

Reply:
(Also, GH trimmed your extra space 😛 - I was staring at it for a while before I kept reading.)

Reply:
FYI, if memory serves I think I've written `%A, %B %e, %Y` in several different repos - may be worth us taking a look through and seeing if we want to change any others.

Reply:
> GH trimmed your extra space
😆